server/utils/files.js

```
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const datastore = './dataStore.json';
+
+// Utility function to read data from file
+function readDataFromFile() {
+  try {
+    const data = fs.readFileSync(datastore, 'utf8');
+    return JSON.parse(data);
+  } catch (error) {
+    console.error('Error reading data from file:', error);
```

I'm not saying this is wrong, but interested to know reasons for letting this function fail silently, rather than just letting it throw?